src/test/java/com/ppojin/util/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,49 @@
+package com.ppojin.util;
+
+import com.ppojin.util.StringAddCalculator;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringAddCalculatorTest {
+
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
+        int result = StringAddCalculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
+
+        result = StringAddCalculator.splitAndSum("");
+        assertThat(result).isEqualTo(0);
+    }
```

Comment:
```suggestion
    @NullAndEmptySource
    @ParameterizedTest
    public void splitAndSum_null_또는_빈문자(String input) {
        int result = StringAddCalculator.splitAndSum(input);
        assertThat(result).isEqualTo(0);
    }
```

`ParameterizedTest`를 사용하면 중복코드를 제거할 수 있습니다.

나머지 테스트케이스에도 적용할 수 있어보여요